application/src/main/java/org/togetherjava/tjbot/commands/code/CodeMessageHandler.java

```diff
@@ -146,8 +154,12 @@ private Button createButtonForAction(CodeAction action, long originalMessageId)
     @Override
     public void onButtonClick(ButtonInteractionEvent event, List<String> args) {
         long originalMessageId = Long.parseLong(args.get(0));
-        CodeAction codeAction = getActionOfEvent(event);
+        if (args.size() >= 3 && DELETE_CUE.equals(args.get(2))) {
```

Comment:
What does this mean?

Reply:
yeah its a bit annoying.

right now, the first arg is the id. the second arg is the label of the code action. i could potentially put "delete" in there, but delete is not a code action, so that would mess up the logic a bit.

so i thought ill just add a third optional argument that indicates any further stuff for extra buttons, deletion in this case.

a comment is definitely the easiest solution out. im out of better ideas

Reply:
comment ✔️

Reply:
If we have more non-code functionalities it can become a switch which would be a lot more readable. This is fine for now!